Jafar_Alhajji/week_01/day_02/js_Homework/main.js

```diff
@@ -0,0 +1,217 @@
+//Homework 
+//What number's bigger?
+
+var numOne = 56;
+var numTwo = 77;
+console.log("Number 1 :" + numOne);
+console.log("Number 2 :" + numTwo);
+if (numOne > numTwo)
+{
+    console.log("Number " + numOne + " is greater than " +numTwo);
+}
+else if (numOne < numTwo)
+{
+    console.log("Number " + numTwo + " is grater than " +numOne);
+}
+else 
+{
+    console.log(numTwo + " = " +numOne);
+}
+
+//--------------
+//Driving Age
+
+var userAge = 16;
+var legalAge = 18;
+
+if (userAge >= legalAge)
+{
+    console.log("Drive away!");
+}
+else 
+{
+    console.log("Sorry, you can't drive yet");
+    console.log("Sorry, you have " + (legalAge - userAge) + " years to wait until you can drive")
+}
+
+//---------------
+
+//The World Translator
+
+var lang = "English";
+
+if (lang === "English")
+{
+    console.log("Hello World");
+}
+else if (lang === "Arabic")
+{
+    console.log("مرحبا بالعالم");
+}
+else 
+{
+    console.log("Hello World in your language");
+}
+
+//------------
+//Suburbs
+var suburb1 = "AL-hasa" ;
+var suburb2 = "Riyadh";
+
+if (suburb1 === "AL-hasa")
+{
+    console.log("because of the team I support");
+}
+else 
+{
+    console.log("because It's the capital");
+}
+
+//-------
+//The Grade Assigner
+var grade = 88;
+
+if (grade <= 100 && grade >=90)
+{
+    console.log("A");
+}
+else if (grade < 90 && grade >=80)
+{
+    console.log("B");
+}
+else if (grade < 80 && grade >=70)
+{
+    console.log("C");
+}
+else if (grade < 70 && grade >=60)
+{
+    console.log("D");
+}
+else if (grade < 60 && grade >=0)
+{
+    console.log("F");
+}
+else
+{
+    console.log("Wrong greade");
+}
```

Comment:
Nice work here and this works perfectly.

Although there is nothing wrong with your solution, one note that might make your life a bit easier. So `else if` is only checked if the `if` ahead of it was evaluated to `false`. What that means is; we don't need to check for _ranges_ like `grade < 90 && grade >=80` because if the `else if` is evaluated then the `if` before it was evaluated to _false_ (ie, grade is not larger than 100). If the first `if` was `true`, then _JS_ will skip all the `else if`s and the `else` after it.

You can read more about this [here](https://www.w3schools.com/Js/js_if_else.asp).